lib/openstax/path_prefixer/action_controller_base_extensions.rb

```diff
@@ -23,4 +24,13 @@ def _compute_redirect_to_location(request, options) #:nodoc:
     end.delete("\0\r\n")
   end
 
+  alias_method :original_url_options, :url_options
+  def url_options
+    # Force the script_name to be in the URL options.  There are some times when
+    # Rails chooses not to, for some inexplicable reasons.  This was added due
+    # to real testing in openstax/accounts; not particularly spec'd here.
+    original_url_options.dup.tap{|options| options[:script_name] = request.script_name}
+  end
+
```

Comment:
I'm assuming this needs to support Rails 4.x?  if not you should probably use `Module.prepend`.  That's the newfangled way of doing this https://www.justinweiss.com/articles/rails-5-module-number-prepend-and-the-end-of-alias-method-chain/

Reply:
Yes, for Accounts.  Good to know about prepend tho.